Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+    <Reference Include="MGCBReferences" />
```

Comment:
Except the MGCB project also includes the Linux platform and as stated, Linux doesn't support the PCL (it used the files as it does now)

Unless there is another way to configure references to be platform specific or exclude Linux?